source/Ubuntu-R.ufo/glyphs/uniE_F_F_D_.glif

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <glyph name="uniEFFD" format="2">
 	<unicode hex="EFFD"/>
-	<advance width="1682"/>
+	<advance width="2502"/>
 	<outline>
-		<component base="zerosuperior" yOffset="-313"/>
-		<component base="period" xOffset="359"/>
-		<component base="eightsuperior" xOffset="605" yOffset="-313"/>
-		<component base="zerosuperior" xOffset="964" yOffset="-313"/>
-		<component base="zerosuperior" xOffset="1323" yOffset="-313"/>
+		<component base="one"/>
+		<component base="period" xOffset="564"/>
+		<component base="zero" xOffset="810"/>
+		<component base="zero" xOffset="1374"/>
+		<component base="zero" xOffset="1938"/>
```

Comment:
This debug glyph is used for double-checking user-provided screenshots during debugging.  Instead of copying the debug glyph some users have actually typed out the version number.  The use of the smaller `superior` figures allows telling apart a screenshot with a real debug symbol, against a manually typed one.

It should be possible to reduce the diff noise by retaining the `yOffset` and `xOffset` as they were.  Ideally any versioning change should be a separate commit from the diff noise.

Reply:
yOffset is tossed because I'm using inferiors instead of shifting superiors.